src/kbe/autofrag.py

```diff
@@ -516,7 +513,6 @@ def autogen(
                 if dist > bond:
                     if inter_dist > dist:
                         inter_dist = dist
-                        inter_idx = ajdx
             inter_dist_ = []
             inter_idx_ = []
             for ajdx, aj in enumerate(coord):
```

Comment:
This part may have changed nontrivially from our old code -- I'll take a closer look soon to determine if this is fine

Reply:
will address in #24; resolving this for now